### ISSUE TITLE
Highlight/Indicate anchor selected entry

### ISSUE THREAD

**Issue by tadman** [rubyapi/rubyapi#730]
When searching for some entries you get "stuck", the page cannot scroll to the top for lack of space, which makes it hard to identify which entry it's actually trying to show.

Example: https://rubyapi.org/3.0/o/exception#method-i-full_message

It'd be nice if there was some JavaScript that could apply a light CSS effect of some kind to indicate which anchor tag was selected.

**Comment by AlexWayfer** [comment]
1. This is default in Web and globally OK.
2. Ruby-Doc.org has the same behavior, example: https://ruby-doc.org/core-3.0.1/Object.html#method-i-untrust

But I can agree that we can make it more noticeable, I was confused too sometimes.

**Comment by tadman** [comment]
Yeah, it's a very old web convention, and while it works fine for things like Wikipedia headers, the sections in Ruby docs can often be quite tiny and cluttered together with others, making it harder to pin down what I'm supposed to be looking at.

**Comment by natematykiewicz** [comment]
We can use the `:target` css selector to style the currently selected anchor.

**Comment by 4lllex** [comment]
Last method height can be set to full page, which would make it scrollable to the top:

```css
section:last-child div[data-controller=method]:last-child {
  min-height: calc(100vh - 74px);
}
```

https://rubyapi.org/4.0/o/string#method-i-valid_encoding-3F

<img width="1321" height="961" alt="Image" src="https://github.com/user-attachments/assets/0f38dbfa-84d6-43df-8133-61e99ba1b673" />
